microsoft-support.md

```diff
@@ -0,0 +1,37 @@
+# Microsoft support for grpc-dotnet
+
+[Microsoft supports grpc-dotnet](https://LINK-TODO) on multiple operating systems, per the [Microsoft Modern Lifecycle](https://support.microsoft.com/help/30881/modern-lifecycle-policy).
+
+Support is provided for the following grpc-dotnet packages:
+* Grpc.AspNetCore
+* Grpc.AspNetCore.Server
+* Grpc.AspNetCore.Web
+* Grpc.AspNetCore.Server.ClientFactory
+* Grpc.AspNetCore.Server.Reflection
+* Grpc.Net.Client
+* Grpc.Net.ClientFactory
+* Grpc.Net.Common
+* dotnet-grpc
+
+Notes:
+* Applications must be using a [currently supported .NET release](https://dotnet.microsoft.com/platform/support/policy).
+* Minimum supported grpc-dotnet version is v2.37.0.
```

Comment:
Will you bump up the min version in future for any reason?

Reply:
Yes. This number is incremented when new .NET versions are released. For example, when .NET 7 is released we'll increment the min version to the version referencing in the new project gRPC template. e.g. .NET 7 template references 2.45.0, which will become the new minimum.

Reply:
(this number should be updated to v2.40.0, which is used by .NET 6 and was released since this document was written)